cmd/error.go

```diff
@@ -1,6 +1,7 @@
 package cmd
 
 import (
+	stdcontext "context"
```

Comment:
I am curious why you chose to use an import alias here?

Reply:
Because there exists a top level `var` named `context` and I didn't want to change that in the current PR :)